prime-dotnet-webapi/Controllers/GisController.cs

```diff
@@ -0,0 +1,210 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using Prime.Services;
+using Prime.Models.Api;
+using Prime.ViewModels.Parties;
+
+namespace Prime.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/[controller]")]
+    [ApiController]
+    public class GisController : ControllerBase
+    {
+        private readonly IGisService _gisService;
+        public GisController(IGisService gisService)
+        {
+            _gisService = gisService;
+        }
+
+        // POST: api/gis
+        /// <summary>
+        /// Creates a new Gis Enrolment
+        /// </summary>
+        [HttpPost(Name = nameof(CreateGisParty))]
+        [ProducesResponseType(typeof(ApiBadRequestResponse), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(StatusCodes.Status200OK)]
```

Comment:
This is slightly incorrect, this endpoint returns a 201 with the GIS in the body, so is more accurately `[ProducesResponseType(typeof(ApiResultResponse<GisViewModel>), StatusCodes.Status201Created)]`